fe/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java

```diff
@@ -0,0 +1,175 @@
+                || columnType.equalsIgnoreCase("BITMAP")
+                || columnType.equalsIgnoreCase("HLL")) {
+            return new StringParser();
+        } else {
+            throw new UserException("unsupported type:" + columnType);
+        }
+    }
+
+    public abstract boolean parse(String value);
+}
+
+class TinyIntParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Short parsed = Short.parseShort(value);
+            if (parsed > 127 || parsed < -128) {
+                return false;
+            }
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class SmallIntParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Short.parseShort(value);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class IntParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Integer.parseInt(value);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class BigIntParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Integer.parseInt(value);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class FloatParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Float.parseFloat(value);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class DoubleParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Double.parseDouble(value);
+        } catch (NumberFormatException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class BooleanParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        if (value.equalsIgnoreCase("true")
+                || value.equalsIgnoreCase("false")) {
+            return true;
+        }
+        return false;
+    }
+}
+
+class DateParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            Date.parse(value);
+        } catch (IllegalArgumentException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class DatetimeParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        try {
+            DateTime.parse(value);
+        } catch (IllegalArgumentException e) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class StringParser extends ColumnParser {
+    @Override
+    public boolean parse(String value) {
+        return true;
```

Comment:
char and varchar type need to check byte length

Reply:
👌